csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -2135,19 +2135,19 @@ and holds the ID for the weakness associated.
 Examples:
 
 ```
-    CWE-79
     CWE-22
     CWE-352
+    CWE-79
```

Comment:
I guess, that should also be:
```suggestion
    CWE-22
    CWE-79
    CWE-352
```

Reply:
The user interface of PRs on sandboxed tablets is kind of multiplicative it seems. Per [the „above“ comment](https://github.com/oasis-tcs/csaf/pull/300#discussion_r671796036) I would appreciate, if we do not make a special case for CWEs - esp. because In my experience the lower numbered CWEs are even more unuseful in day to day developer support than the higher numbered ones 🏃🏽‍♂️ 🧯

Reply:
As above: I withdraw the comment ;-)